api/RTCIceCandidatePairStats.json

```diff
@@ -1460,10 +1460,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "11"
```

Comment:
This (`totalRoundTripTime`) is also in https://github.com/WebKit/WebKit/commit/ed457da32eb3b03501a8741588c14245e4974f6d, and in `RTCStatsReport.idl` currently.